src/schema/index.ts

```diff
@@ -1,12 +1,12 @@
-import { gql } from 'apollo-server'
+import { gql } from 'apollo-server';
 
 const Schema = gql`
   type User {
-    id: ID!
+    _id: ID!
     role: String!
     email: String!
     password: String!
-    profile: Profile!
+    profile_id: String!
```

Comment:
This is not just an id, but a children of the user schema that is resolved below.
You can keep the name `profile` to avoid confusion